test/components/elements/AutoCompleteMenuTest.js

```diff
@@ -0,0 +1,227 @@
+'use strict'
+import chai from 'chai'
+import spies from 'chai-spies'
+chai.use(spies)
+const expect = chai.expect
+
+import React from 'react'
+import AutoComplete from 'material-ui/AutoComplete'
+
+import createComponent from 'test/helpers/shallowRenderHelper'
+import AutoCompleteMenu, { onNewRequest } from 'components/elements/AutoCompleteMenu'
+import AddMockDispatch from 'test/helpers/mockDispatch'
+
+
+const mockProps = {
+    label: 'testLabel',
+    dataConfigText: 'textInput',
+    dataConfigValue: 'valueInput',
+    menuOptions: ['TEST1', 'TEST2'],
+    searchText: '',
+    onUpdateInput: () => null,
+    selectMethod: () => 42
+}
+
+const setup = (mockProps) => {
+
+    return createComponent(AutoCompleteMenu, mockProps)
+}
+
+describe('AutoCompleteMenu', function() {
+  describe('Component', function() {
+    describe('when it receives all required information', function () {
+      beforeEach(function () {
+          this.output = setup(mockProps)
+      })
+
+      it('should return a div', function () {
+          expect(this.output.type).to.equal('div')
+      })
+
+      it('the div should contain an AutoComplete', function () {
+          expect(this.output.props.children.type).to.equal(AutoComplete)
+      })
+
+      it('the Autocomplete should have the label we passed in', function () {
+          expect(this.output.props.children.props.floatingLabelText).to.equal(mockProps.label)
+      })
+
+      it('the Autocomplete should have the searchText we passed in', function () {
+          expect(this.output.props.children.props.searchText).to.equal(mockProps.searchText)
+      })
+
+      it('the Autocomplete should have the menuOptions we passed in', function () {
+          expect(this.output.props.children.props.dataSource).to.equal(mockProps.dataSource)
+      })
+
+      it('the Autocomplete should a data config object using info we passed in', function () {
+          expect(this.output.props.children.props.dataSourceConfig).to.deep.equal({ text: mockProps.dataConfigText, value: mockProps.dataConfigValue })
+      })
+
+      it('the Autocomplete should have an onUpdateInput fx just like the fx we passed in', function () {
+          expect(this.output.props.children.props.onUpdateInput()).to.equal(mockProps.onUpdateInput())
+      })
+    })
+  })
+
+  describe('onNewRequest', function() {
+    const selectMethod = (input) => input
+    const clearMethod = () => console.log('')
```

Comment:
is this clearMethod like a no-op? the test runner may not necessarily have a console object, and even if it does, this may have side effects.